pangloss/data_fetcher.py

```diff
@@ -0,0 +1,71 @@
+from os import mkdir, path, remove
+from traceback import print_exc
+
+from requests import get
+
+from pangloss import CATALOG_EXAMPLE, DATA_DIR, GAMMA_1_FILE, GAMMA_2_FILE, GUO_FILE, \
+    HALO_MASS_REDSHIFT_CATALOG, KAPPA_FILE, KAPPA_EXAMPLE, PANGLOSS_DIR
+
+DEMO_DATA_URL = "http://www.slac.stanford.edu/~pjm/hilbert"
+CALIB_DATA_URL = "http://www.ast.cam.ac.uk/~tcollett/Pangloss/calib"
+
+
+def setup_data():
```

Comment:
Does this file not need to be imported in the `__init__.py` so that you can call `pangloss.setup_data()` from the notebooks? I think it probably does...